examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>so we can save this as <code>item_attributes</code> if you want like below:</p><p><code></code><code></p><p>item_features.to_parquet(</p><p>&nbsp;&nbsp;os.path.join(feature_repo_path, "data", "item_attributes.parquet")</p><p>)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>